### DISCUSSION TITLE
so-aptcacherng ----------- [ MISSING ]

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#1572] [Unsupported Versions] [answered]
If I chose "OPEN" to the question "How would you like to download OS package updates for your grid?" during installing, the aptcacherng won't be installed.
But so-status shows so-aptcacherng is missing.
Can you fix this?

Install type. STANDALONE

**Comment by ghost** [comment, marked as answer]
By choosing **open** you have opted to not need aptcatcherng because you are not caching the repos locally. You are letting your sensor go online to pull down latest updates there. It likely never gets installed when you go this route.

**Reply by TOoSmOotH** [reply]
Yea it should not show up as missing. I will put in an issue.
